extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.css

```diff
@@ -20,6 +20,20 @@
   transform: scale(1.0) translateX(-100%) translateY(200%) !important;
 }
 
+/**
+ * amp-story-auto-ads tags should never be visible. keep them hidden.
+ */
+amp-story-auto-ads {
+  /* Fixed to make position independent of page other elements. */
+  position: fixed !important;
+  top: 0 !important;
+  left: 0 !important;
+  width: 1px !important;
+  height: 1px !important;
+  overflow: hidden !important;
+  visibility: hidden;
+}
+
```

Comment:
LGTM. Thanks!

Reply:
@calebcordry could you help to patch the change locally and make sure it works for amp-story-auto-ads (i think it does, just to be super cautious)

Reply:
I checked a few local test files, they look good.